activesupport/lib/active_support/core_ext/date/calculations.rb

```diff
@@ -1,12 +1,12 @@
-require 'date'
```

Comment:
@matthewd, you mean in case the require for `'active_support/core_extdate/zones'` is removed?